### PR TITLE
Break elasticsearch queries to several

### LINKED ISSUE #3003: [production] KPIs not updated to Dashboard overview
# Steps to reproduce
1. Login to apinf.io (e.g. as admin)
2. Go to dashboard
3. See the overview for APInf Catalog REST API
4. Open the detailed view for the same API and compare

# Expected result
See the same figures for Number of requests, Response time(median) and Users

# Actual result
Overview contains 0 for Number of requests, Response time(median) and Users, while Detailed view shows the actual figures

#Screenshots
3003_production dashboard overview

3003_production dashboard detailed view

### LINKED ISSUE #3071: Dashboard: Break single ElasticSearch query to multiple queries
# User story
As an API owner or administrator, I do not want to wait for the entire page to load before I can see any data on the dashboard Details view.

# Description
In the current Details view implementation, a single ElasticSearch query for the selected timeframe. This means that data for all charts and figures needs to be returned before anything gets shown on the dashboard. Hence with APIs with large amount of calls, users might be looking at the spinner for 30s.

# Goal
Single ES query is broken into multiple queries, perhaps query per chart.

### PR DESCRIPTION
**Pull request by marla-singer**
Closes #3071 
Closes #3003 

## Changes
- Replace all charts under folder `/client/charts`
- Replace all tables under folder `/client/tables`
- Break existed queris to Elasticsearch to several
- Create three general queries for Dashboard page and Analyticspage to get data for Overview charts, total statistics and response status codes
- Create separated queries to get data for timeline charts, errors statistics, users statistics

## Developer checklist
This checklist is to be completed by the PR developer:

- [ ] Alternative solutions were compared/discussed before writing code
    - [ ] trade-offs with this solution are considered acceptable
- [ ] Code in this PR adheres to the [project styleguide](CONTRIBUTING.md)
- [ ] This pull request does not decrease project test coverage
- [ ] If the code changes existing database collection(s), migration has been written
- [ ] If UI texts are added or changed, all texts are internationalized

## Reviewer checklist
Reviewed by: @55

This list is to be completed by the pull request reviewer:
- [ ] Code works as described/expected
- [ ] Code seems to be error free
    - [ ] no browser console errors visible
    - [ ] no server console errors visible
    - [ ] passes CI build
- [ ] Code is written in a way that promotes maintainability
    - [ ] easy to understand
    - [ ] well organized
    - [ ] follows project coding standards and conventions
    - [ ] well documented

### CODE REVIEW

**Comment by marla-singer** [comment]
@matleppa Ready for review. If you have some question about code then write it. I will try to add comments into code 

About enhancement/bugs also write. I will open the follow-up issues to improve.

File: apinf_packages/dashboard/client/analytic/body/body.js [hunk 1]
```diff
@@ -15,78 +15,330 @@ import { FlowRouter } from 'meteor/kadira:flow-router';
...
+  errorsStatisticsResponse () {
+    const instance = Template.instance();
+
+    // Return boolean value of available errorsStatistics response
+    return instance.errorsStatisticsResponse.get();
   },
-  dataAvailable () {
-    const elasticsearchData = Template.instance().elasticsearchData.get();
+  errorsStatisticsData () {
+    // Get data of rrorsStatistics response
```
Comment (matleppa) [inline apinf_packages/dashboard/client/analytic/body/body.js]: Very minor typo rrorsStatistics

File: apinf_packages/dashboard/client/view/summary_statistics/summary.html [hunk 2]
```diff
@@ -31,49 +31,49 @@
...
+              {{# with comparisonData proxyBackend.frontendPrefix  }}
+                <div class="{{ arrowDirection 'requests' }}"></div>
+                <span class="{{ textColor 'requests' }}">
+                  {{ percentages 'requests' }}
+                </span>
+              {{/ with }}
+            </div>
+            <div class="col-xs-3">
+              <span class="response-time">
+                {{ getCount proxyBackend.frontendPrefix  'time' }} ms
```
Comment (matleppa) [inline apinf_packages/dashboard/client/view/summary_statistics/summary.html]: Extra space before 'time'

File: apinf_packages/dashboard/client/view/summary_statistics/summary.html [hunk 3]
```diff
@@ -31,49 +31,49 @@
...
+              {{# with comparisonData proxyBackend.frontendPrefix  }}
+                <div class="{{ arrowDirection 'time' }}"></div>
+                <span class="{{ textColor 'time' }}">
+                  {{ percentages 'time' }}
+                </span>
+              {{/ with }}
+            </div>
+            <div class="col-xs-3">
+              <span class="unique-users">
+                {{ getCount proxyBackend.frontendPrefix  'users' }}
```
Comment (matleppa) [inline apinf_packages/dashboard/client/view/summary_statistics/summary.html]: Extra space before 'users'.

**Comment by marla-singer** [comment]
@matleppa Fixed the requested changes
@55 I added "min-height".What about the spinner in the center of the page - it relates to subscription and runs before to display Charts placeholders

@matleppa @55 If everything is okay please don't merge it yet. I wanna squash commits into one before merging

**Comment by matleppa** [comment]
Product owners wanted this to be merged in spite of missing commit squash.

**Pull request merged**